pkg/cli/admin/groups/sync/sync.go

```diff
@@ -417,6 +417,8 @@ func (o *SyncOptions) Run() error {
 			},
 		}
 		for _, item := range openshiftGroups {
+			item.APIVersion = "user.openshift.io/v1"
```

Comment:
The fix is reasonable, but how about making this where the `Group` object is actually created, iow.: https://github.com/openshift/oc/blob/f749845d71de73620f5e66553e0dd5abeb7efede/pkg/helpers/groupsync/groupsyncer.go#L151

Reply:
that's actually what I originally did in this PR, but it still didn't insert these fields automatically.

I can switch back to creating the `Group` object, but we still need to manually add these fields (or figure out why they're not showing up)